includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -66,10 +66,34 @@
                     $vmwVmDisplayName = $xml->name;
                     $vmwVmGuestOS     = '';
                     // libvirt does not supply this
-                    $vmwVmMemSize = ($xml->currentMemory / 1024);
-                    exec($config['virsh'].' -c '.$uri.' domstate '.$dom_id, $vm_state);
+                    exec($config['virsh'].' -rc '.$uri.' domstate '.$dom_id, $vm_state);
                     $vmwVmState = ucfirst($vm_state[0]);
-                    $vmwVmCpus  = $xml->vcpu;
+
+                    $vmwVmCpus = null;
+                    $vmwVmMemSize = null;
+                    if ($xml->xpath("//sysinfo/system/entry[@name='manufacturer'][.='oVirt']")) {
+                        d_echo("Parsing information for $vmwVmDisplayName with oVirt quirx");
```

Comment:
I tend to agree with @mykaul after reading the specification. I think it is a better approach to use the values of `memory` (with unit conversion) and the `current` attribute for `vcpu` for the amount of CPUs as they seem to follow the standard. If this attribute is absent, we can fall back on the value of the `vcpu` element itself. I guess I can work on that tonight.